command/rootchain/premine/premine.go

```diff
@@ -128,27 +126,6 @@ func runCommand(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("approve transaction failed on block %d", receipt.BlockNumber)
 	}
 
-	premineFn := &contractsapi.AddGenesisBalanceCustomSupernetManagerFn{
```

Comment:
This command is now incomplete. Since it will only be used for bridge, when we have non-mintable native `ERC20` token, it should send a transaction to the `RootERC20Predicate` to freeze the premined amount of tokens of sender on rootchain. This will mean that we need to have a premine function on the `RootERC20Predicate` contract.